app/controllers/customers_controller.rb

```diff
@@ -1,6 +1,6 @@
 class CustomersController < ApplicationController
+   before_action :set_customer, only: [:show, :edit, :update]
   def show
-  @customer = Customer.find(params[:id])
   @chintus = @customer.chintus
```

Comment:
@Chintu766  why do we need **@chintus** object here? It is the Customers controller right? then we should have customer objects in show API right?